src/console_reporter.cc

```diff
@@ -113,4 +121,16 @@ void ConsoleReporter::PrintRunData(const Run& result) {
               result.report_label.c_str());
 }
 
+TimeUnitMultiplier ConsoleReporter::getTimeUnitAndMultiplier(TimeUnit unit) {
```

Comment:
This comment is unclear to me, since csv and json reporter currently don't display the time unit. Nevertheless I moved the function to `BenchmarkReporter` as you suggested.